### ISSUE TITLE
How can I use metrics such as recall and F1 score?

### ISSUE THREAD

**Issue by Tess314** [google/mediapipe#4832]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Windows 10

### Python Version

3.10

### Task name (e.g. Image classification, Gesture recognition etc.)

Image classification

### Describe the actual behavior

Ability to produce precision, recall and F1 scores

### Describe the expected behaviour

Only able to produce loss and accuracy metrics

### Standalone code/steps you may have used to try to get what you need

```shell
# model.evaluate only allows for loss and accuracy
# how to get precision, recall, and f1 score without predict function?

loss, acc = model.evaluate(test_data)
print(f'Test loss:{loss}, Test accuracy:{acc}')
```

**Comment by joezoug** [comment]
Hi @Tess314,

This has been a backlogged feature request we've been meaning to implement. I'll raise this issue internally and try to have a fix out soon! In the meantime you can use this code snippet to add custom metric functions after you've called `create`.

```
metric_functions = model._metric_functions + [tf.keras.metrics.F1Score(), tf.keras.metrics.Precision(), tf.keras.metrics.Recall()]
model._model.compile(
    optimizer=model._optimizer,
    loss=model._loss_function,
    metrics=metric_functions,
)

metrics = model.evaluate(test_data)
print(metrics)
```

**Comment by Tess314** [comment]
Thank you so much, that worked great :)
